### ISSUE TITLE
Potential bug? Keycard or rectangular keycard?

### ISSUE THREAD

**Issue by firstdeathmaker** [Microsoft/TextWorld#24]
I generated a new game and started it. But at a certain point, I get stuck, because I have a keycard that I have to use to open a portal. I can't use that keycard, because I have another, rectangular keycard in my inventory and the game doesn't understand which one I want to use:

> examine gateway
it's a solid portal It is closed.

> open gateway
You have to unlock the gateway with the key first.

> open gateway with keycard
Which do you mean, the keycard or the rectangular keycard?

> open gateway with the non rectangular keycard
You can't see any such thing.

> open gateway with the rectangular keycard
That doesn't seem to fit the lock.`

**Comment by MarcCote** [comment]
Do you have the command you used to generate the game? Also, what's the Global seed (it should be printed out by the tw-make script)?

**Comment by firstdeathmaker** [comment]
I used the example command:
`tw-make custom --world-size 5 --nb-objects 10 --quest-length 5 --output gen_games/`

I don't have the seed of the game anymore, but I got another seed from a game that shows that you sometimes get a keycard (without any further attributes) from the beginning. The seed is `53958`, although in that game, you don't need to open any doors and therefore the keycard is useless. I'll try to find another seed that shows that bug, but I'm not sure how often that happens.
